src/App.svelte

```diff
@@ -61,8 +63,16 @@
         medicareTaxes = fica[1];
 
         // Update tax info
-        totalTaxes = incomeTaxes + socialSecurityTaxes + medicareTaxes;
+        totalTaxes = incomeTaxes + socialSecurityTaxes + medicareTaxes - nonRefundableTaxCredit;
+        if (totalTaxes < 0) {
+            totalTaxes = 0;
+        }
         afterTaxIncome = grossIncome - totalTaxes;
+        totalTaxes -= refundableTaxCredit;
```

Comment:
Also, you surrently have a bug where changes to refundable tax credit don't affect after tax income:
![image](https://user-images.githubusercontent.com/55931391/234510276-6f1bb0c5-c984-4de4-925f-d7829ee83103.png)

I think it's due to these lines here because you're updating total taxes with the non-refundable credits after you updated afterTaxIncome. Instead you should switch the positions of `totalTaxes -= refundableTaxCredit;` and `afterTaxIncome = grossIncome - totalTaxes;` so `afterTaxIncome` gets updated last .

Reply:
All refundable tax credit is now refunded and shown in after tax income.